app-tests/nek5000/nek5000_check.py

```diff
@@ -0,0 +1,47 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+@rfm.simple_test
+class nek5000_check(rfm.RegressionTest):
+    valid_systems = ['lumi']
```

Comment:
This will make the tests run for all partitions.

Reply:
Fixed with 6181b10